doc/source/guide/geometry_guide.rst

```diff
@@ -0,0 +1,286 @@
+Function spaces (discretized or continuous) in ODL are, up to a few exceptions, defined on rectangular domains.
+Such spaces have a relatively simple structure and can be represented and manipulated very efficiently.
+Therefore ODL does not represent data directly as functions on manifolds, but rather as functions on a *coordinate domain* that paremetrizes the manifold.
+
+For instance, in the 2D parallel beam example above, a unit vector :math:`\theta \in \mathbb{S}^1` can be parametrized by an angle :math:`\varphi \in [0, 2\pi)`, and a vector :math:`v` on the line :math:`\theta^\perp` by a single number :math:`s \in \mathbb{R}`.
+Such a representation additionally requires a *convention* for a translation between coordinates :math:`(\varphi, s)` and points :math:`(\theta, v)` on the manifold, i.e., a map between the coordinate domain and the data manifold.
+Such a map is usually called a *parametrization* or `chart`_ of the manifold.
+
+In our example, we could thus redefine the ray transform as a map between Euclidean function spaces like this:
+
+    .. math::
+        & \mathcal{R} :  L^2(\Omega) \to L^2(I \times D),
+
+        & \mathcal{R}(f)(\varphi, u) := \int_{\mathbb{R}} f\big( u\theta(\varphi - \pi/2) + t\theta(\varphi) \big)\, \mathrm{d}t.
+
+Here, :math:`I \subset [0, 2\pi)` and :math:`D \subset \mathbb{R}` are intervals and
+
+    .. math::
+        & (\varphi, u) \mapsto \big( \theta(\varphi), u \theta(\varphi - \pi/2)\big),
+
+        & \theta(\varphi) := (-\sin\varphi, \cos\varphi)
+
+is a parametrization of the data manifold.
+
+.. figure:: figures/parallel2d_geom.svg
+    :width: 75%
+    :align: center
+
+    Parametrization of lines in 2D parallel beam geometry.
+
+
+
+.. _manifold: https://en.wikipedia.org/wiki/Manifold
+.. _chart: https://en.wikipedia.org/wiki/Manifold#Charts
+
+
+
+Geometries in ODL
+=================
+The `RayTransform` in ODL is an `Operator` between `DiscreteLp` type discretized function spaces defined on rectangular domains.
+These rectangular regions either represent a reconstruction region ("volume") or an interval product of parameters describing the data.
+The missing component, which is the mapping from coordinates to points on the data manifold, is encoded in the `Geometry` class and its subclasses as described in the following.
+
+
+The `Geometry` class
+--------------------
+All ODL geometries derive from the abstract `Geometry` class that provides a basic structure.
+Most attributes are intended to query for geometric information, e.g., source and detector positions and their orientations.
+See the documentation of `Geometry` and `Detector` for details on the API.
+
+
+Geometric definitions and conventions
+-------------------------------------
+The initial state of the system corresponds to motion parameters :math:`m = 0`.
+In many geometries, the motion parameters are given by a single angle of rotation, but they can refer to any encoding of the motion of the acquisition system.
```

Comment:
That wouldn't really be so different from the "single angle of rotation" case. It's 2 angles, okay, but that's not so exciting.
Anyway, I've tried to put up an example using `Parallel3dEulerGeometry`, and it turned out that the backend code was not ready for it. A fix is in, working on the example.